react/components/molecules/chat-message-placeholder/chat-message-placeholder.js

```diff
@@ -0,0 +1,122 @@
+import { StyleSheet, View } from "react-native";
+import { PlaceholderContext, isTabletSize } from "ripe-commons-native";
+import LinearGradient from "react-native-linear-gradient";
+import PropTypes from "prop-types";
+
+export class ChatMessagePlaceholder extends PureComponent {
+    static get propTypes() {
+        return {
+            variant: PropTypes.string,
+            style: PropTypes.any,
+            itemStyle: PropTypes.any
+        };
+    }
+
+    static get defaultProps() {
+        return {
+            variant: undefined,
+            style: {},
+            itemStyle: {}
+        };
+    }
+
+    _style() {
+        return [styles.chatMessage, this.props.style];
+    }
+
+    render() {
+        return (
+            <PlaceholderContext.Consumer>
+                {({ placeholderLocations, placeholderColors }) => (
+                    <View style={this._style()}>
+                        <View style={styles.messageLeft}>
+                            <LinearGradient
+                                style={styles.avatar}
+                                angle={90}
+                                useAngle={true}
+                                colors={placeholderColors}
+                                locations={placeholderLocations}
+                            />
+                        </View>
+                        <View style={styles.messageRight}>
+                            <View style={styles.header}>
+                                <LinearGradient
+                                    style={styles.username}
+                                    angle={90}
+                                    useAngle={true}
+                                    colors={placeholderColors}
+                                    locations={placeholderLocations}
+                                />
+                                <LinearGradient
+                                    style={styles.date}
+                                    angle={90}
+                                    useAngle={true}
+                                    colors={placeholderColors}
+                                    locations={placeholderLocations}
+                                />
+                            </View>
+                            <View style={styles.content}>
+                                <LinearGradient
+                                    style={styles.text}
+                                    angle={90}
+                                    useAngle={true}
+                                    colors={placeholderColors}
+                                    locations={placeholderLocations}
+                                />
+                            </View>
+                        </View>
+                    </View>
+                )}
+            </PlaceholderContext.Consumer>
+        );
+    }
+}
+
+const styles = StyleSheet.create({
+    chatMessage: {
+        width: "100%",

```

Comment:
is this really needed while having `flex: 1` below?

Reply:
Nope, it was not needed. Not even `flex: 1` because it's the default.

Reply:
I am not really sure about the `flex: 1` removal, because it's not the default value ( [see here](https://yogalayout.com/docs/flex))